test/utilities/terminalView.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import clipboard from 'clipboardy'
+import {
+  TerminalView,
+  Workbench
+} from 'wdio-vscode-service';
+import {
+  CMD_KEY
+} from 'wdio-vscode-service/dist/constants';
+import {
+  log,
+  pause
+} from './miscellaneous';
+
+export async function getTerminalView(workbench: Workbench): Promise<TerminalView> {
+  const bottomBar = await workbench.getBottomBar();
+  const terminalView = await bottomBar.openTerminalView();
+  await pause(5);
+
+  return terminalView;
+}
+
+export async function getTerminalViewText(terminalView: TerminalView, seconds: number): Promise<string> {
+  for (let i=0; i<seconds; i++) {
+    await pause(1);
+
+    // const terminalText = await terminalView.getText();
```

Comment:
Can this be removed?

Reply:
I'd prefer to keep it in.  `terminalView.getText()` _should_ be used here, but due to a bug it isn't used and the code that follows it a work around.  My hope i that if `terminalView.getText()` is fixed, that the line will be uncommented and  and the code that follows it be removed.  I added some additional comments in the code.  How about what's there now?

Reply:
Looks good - thanks!